### PR TITLE
Feedback fix

### LINKED ISSUE #52: Fix feedback counter and provide a standalone feedback action

### PR DESCRIPTION
**Pull request by Razz4780**
Closes #52

Marketplace review prompt is now displayed after each 100 runs and can be disabled.
Feedback action (google forms) was already there

### CODE REVIEW

File: src/api.ts [hunk 1]
```diff
@@ -348,23 +348,24 @@ export class MirrordAPI {
...
-  globalContext.globalState.update(FEEDBACK_COUNTER, counter + 1);
+  const previousRuns = parseInt(globalContext.globalState.get(FEEDBACK_COUNTER) ?? '0');
+  const currentRuns = previousRuns + 1;
 
-  if (counter >= FEEDBACK_COUNTER_REVIEW_AFTER) {
+  globalContext.globalState.update(FEEDBACK_COUNTER, currentRuns);
+
+  if ((currentRuns % FEEDBACK_COUNTER_REVIEW_AFTER) === 0) {
     new NotificationBuilder()
-      .withMessage(`Enjoying mirrord? Don't forget to leave a review! Also consider giving us some feedback, we'd highly appreciate it!`)
```
Comment (eyalb181) [inline src/api.ts:360]: Sorry, lets leave the feedback part in, my mistake

Comment (Razz4780) [reply src/api.ts:360]: Np, restored

File: src/status.ts [hunk 2]
```diff
@@ -6,12 +6,12 @@ import { NotificationBuilder } from './notification';
 
 export class MirrordStatus {
```
Comment (eyalb181) [inline src/status.ts:7]: What are the changes in this file for?

Comment (Razz4780) [reply src/status.ts:7]: Wanted to use the already existing constant for feedback command id, but the property was not static and we don't store any global instance of `MirrordStatus`. So I made all command ids static for consistency

**Review by eyalb181 — APPROVED** [review]

**Pull request merged**